.pylint2rc

```diff
@@ -93,7 +93,7 @@ evaluation=10.0 - ((float(5 * error + warning + refactor + convention) / stateme
 [FORMAT]
 
 # Maximum number of characters on a single line.
-max-line-length=80
+max-line-length=100
```

Comment:
https://bravesoftware.slack.com/archives/C7VLGSR55/p1659417448741579

Reply:
let's revert once you got a green build and no other pylint issues

Reply:
This is already green in CI. But if we revert, then any change to this file will produce 3 pylint warnings in any PR that touches it. I think this limit is worth reconsidering anyhow.

Reply:
to be honest I'd the comment to exclude the pylons rule for every longer line. this also gives us indication how often this occurs.

Reply:
We could also do that.

That said, I think this is bad for code readability. IMO there's a price to be paid for every line a human has to read to understand what's going on. We can only hold so much in our minds before we lose track of things and the comments aren't just automatically filtered out, they do take some amount of brain power/capacity to process. And those comments contribute nothing towards explaining what the code does, they're pure noise to a human, only serve to distract.

Reply:
Ok, I'll leave this discussion for another PR. Fixed in this one via a script-wide ignore for the line-too-long check.